src/main/java/org/wildfly/security/auth/realm/ldap/OtpCredentialLoader.java

```diff
@@ -128,8 +129,8 @@ public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential
                 Password password = passwordFactory.generatePassword(new OneTimePasswordSpec(
                                 CodePointIterator.ofString((String) hashAttribute.get())
                                         .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
-                                CodePointIterator.ofString((String) seedAttribute.get())
-                                        .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
```

Comment:
@honza889 @dmlloyd Based on your inputs, I updated the FileSystemSecurityRealm according, now we are reading seed and hash values with padding optional. Writes were already using padding.

There are no more pending things to do here on my side.